Source/SpacePort/DBAssets/DbModels/SpaceshipModel.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using System.Threading.Tasks;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace SpacePort
+{
+    public class SpaceshipModel
+    {
+        [Key]
+        public int SpaceshipID { get; private set; }
+        [MaxLength(50)]
+        public string Name { get; set; }
+        public double Length { get; set; }
+
+        //Relationships
+        public int? PersonID { get; set; }
+        [ForeignKey("PersonID")]
+        public PersonModel Person { get; set; }
+
+        //Methods
+        public static SpaceshipModel CreateModelFromAPI(ApiDataFetch dataFetch, int searchIndex)
```

Comment:
Good to inject the ApiDataFetch 🥇 
I'm unsure what this methods does (based on the naming).

Reply:
Namngivningen hade kanske kunnat vara bättre. Det är en metod som finns statiskt i klassen för att tillåta skapandet av ett databas-modell-objekt genom att hämta data direkt från API. Vi har senare resonerat att vi inte vill att detta ska vara möjligt och denna metod skulle förmodligen ha tagits bort. Resonemanget var snarare att modeller endast ska användas mot databasen och att API data inte används direkt mot databasen.